data/strings/strings.txt

```diff
@@ -22591,6 +22594,7 @@
     ru:few = %d метки
     ru:one = %d метка
     ru:other = %d меток
+    ru:zero = %d меток
```

Comment:
Does WL require it?
Because its not necessary, its the same form as many/other.
Same for uk and pl, AFAIK.

Reply:
If WL doesn't require it then its better to avoid having redundant strings

Reply:
> Does WL require it?

It doesn't. I can remove.

Reply:
Removed.